tests/test_azure.py

```diff
@@ -26,9 +26,9 @@
 class AZFSTest(test.TestCase):
```

Comment:
That was from the past code but I think we can use `tf.test.TestCase` as well.

Also, `tf.io.gfile` can be used directly. So at the beginning of the file:
```
from tensorflow.python.platform import test
from tensorflow.python.platform import gfile
```

can be replaced with:
```
import tensorflow as tf
```

Reply:
addressed in the commit